template/.github/workflows/ios_deploy_staging_to_firebase.yml

```
@@ -0,0 +1,52 @@
+name: ios-deploy-to-staging
+on:
+  # Trigger the workflow on push action
+  push:
+    branches:
+      - develop
+
+jobs:
+  build_and_upload_staging_app_to_firebase:
+    name: Build And Upload iOS Staging Application To Firebase
+    runs-on: macOS-latest
+    env:
+      TEAM_ID: ${{ secrets.TEAM_ID }}
+      FASTLANE_USER: ${{ secrets.FASTLANE_USER }}
+      FASTLANE_PASSWORD: ${{ secrets.FASTLANE_PASSWORD }}
+      FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD: ${{ secrets.FASTLANE_APPLE_APPLICATION_SPECIFIC_PASSWORD }}
+      FASTLANE_SESSION: ${{ secrets.FASTLANE_SESSION }}
+      MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
+      KEYCHAIN_PASSWORD: ${{ secrets.KEYCHAIN_PASSWORD }}
+    steps:
+    - uses: actions/checkout@v1
```

please set step name

```suggestion
    - name: Checkout
      uses: actions/checkout@v3
```